fabfile.py

```diff
@@ -216,31 +234,36 @@ def start(c, workload):
 @task
 def stop(c):
 
+    skip_services = False
```

Comment:
I don't think this is an issue to stop services even for `test/format` because they won't be running so it ends up being a no op, what do you think?

Reply:
This needs to be done otherwise we have senseless logs being printed - and I think it is good practise to not do the work that is unnecessary. What do you think?

Reply:
I agree. Do you think we could then encapsulate whatever is dedicated to starting/stopping services in functions?